src/components/editor/Bindings/Store/create.ts

```diff
@@ -166,49 +139,42 @@ const getInitialState = (
 
     setEditModeEnabled: (value) => {
         set(
-            produce((state: BindingsEditorState) => {
-                state.editModeEnabled = value;
-            }),
+            (state) => ({ ...state, editModeEnabled: value }),
```

Comment:
Common for this PR. Removing `immer` when it was not 100% needed or was REALLY complex to remove.